src/content/topics/sdk/concepts/version-notifications.mdx

```diff
@@ -66,14 +66,13 @@ Here's how:
 </CodeTabItem>
 </CodeTabs>
 
-## Getting notifications on the LaunchDarkly web application
+## Getting notifications in the LaunchDarkly web application
 
-SDK notifications are available on the [Flags list](/home/organize/flags-list). You receive notifications when
-SDK versions near their end of life, are past their end of life, and when major versions are available.
-The notifications are based on SDK usage within each LaunchDarkly environment.
+Within the LaunchDarkly application, you'll automatically receive notifications when SDK versions are nearing end of life (one month prior) and again when they are past end of life. The notifications are based on SDK usage within each LaunchDarkly environment. See the full list of [LaunchDarkly's supported SDK versions](https://docs.launchdarkly.com/sdk/concepts/versioning#supported-versions). 
```

Comment:
Minor URL change so that the link works correctly on our staging docs site too.

```suggestion
Within the LaunchDarkly application, you'll automatically receive notifications when SDK versions are nearing end of life (one month prior) and again when they are past end of life. The notifications are based on SDK usage within each LaunchDarkly environment. See the full list of [LaunchDarkly's supported SDK versions](/sdk/concepts/versioning#supported-versions). 
```